intern/remote/boss/remote.py

```diff
@@ -866,13 +890,14 @@ def get_cutout(self, resource, resolution, x_range, y_range, z_range, time_range
                 z_range (list[int]): z range such as [10, 20] which means z>=10 and z<20.
                 time_range (optional [list[int]]): time range such as [30, 40] which means t>=30 and t<40.
                 id_list (optional [list[int]]): list of object ids to filter the cutout by.
-                no_cache (optional [boolean]): specifies the use of cache to be True or False. 
+                no_cache (optional [boolean]): specifies the use of cache to be True or False.
 
             Returns:
                 (numpy.array): A 3D or 4D (time) numpy matrix in (time)ZYX order.
 
             Raises:
                 requests.HTTPError on error.
             """
-
+            if isinstance(resource, str):
+                resource = self.parse_bossURI(resource)
```

Comment:
Can you update the doctoring to indicate that resource can also be a URI as you describe in `parse_bossURI`?

Reply:
Just updated [here](https://github.com/jhuapl-boss/intern/pull/32/files#diff-52eff90a39a42536cd56c92770a07c38R886)!